ginga/misc/plugins/MultiDim.py

```diff
@@ -493,6 +532,46 @@ def redo(self):
 
         self.w.hdu.set_enabled(len(self.fits_f) > 0)
 
+    def save_slice_cb(self):
+        target = SaveDialog(title='Save slice', extfilter='PNG image (*.png)').get_path()
+        if target:
+            hival = self.fitsimage.get_cut_levels()[1]
+            plt.imsave(target, self.curr_slice, vmax=hival, cmap=plt.get_cmap('gray'), origin='lower')
+            self.fv.showStatus("Successfully saved slice")
```

Comment:
Suggest:
```python
image = self.fitsimage.get_image()
curr_slice = image.get_data()
```
Creating instance variables that hold on to potentially large arrays makes me nervous...